src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -116,6 +116,13 @@ def test_properties_rw_base():
                 (log.Level.TRACE, "Level.TRACE", 4),
             ),
         ),
+        (
```

Comment:
Please, add direct import also [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/properties/intel_cpu/__init__.py)
[for example](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/properties/intel_auto/__init__.py) other enum in other submodule

Reply:
Updated, please take a look.